### ISSUE TITLE
Segmentation fault (core dumped) when running stable diffusion demo on A10

### ISSUE THREAD

**Issue by brucelil** [NVIDIA/TensorRT#2886]
## Description

I follow this guide https://github.com/NVIDIA/TensorRT/tree/release/8.6/demo/Diffusion to test with Nvidia A10  step by step， but  it was Segmentation fault when  Building TensorRT engine for onnx/unet.opt.onnx: engine/unet.plan，any ideas?

some logs:

UNet: fold constants .. 4208 nodes, 6924 tensors, 3 inputs, 1 outputs
UNet: shape inference .. 4208 nodes, 6924 tensors, 3 inputs, 1 outputs
UNet: finished .. 4208 nodes, 6924 tensors, 3 inputs, 1 outputs
Exporting model: onnx/vae.onnx
Downloading (…)on_pytorch_model.bin: 100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 335M/335M [01:58<00:00, 2.82MB/s]
Downloading (…)main/vae/config.json: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 611/611 [00:00<00:00, 157kB/s]
========== Diagnostic Run torch.onnx.export version 1.14.0a0+44dac51 ===========
verbose: False, log level: Level.ERROR
======================= 0 NONE 0 NOTE 0 WARNING 0 ERROR ========================

Generating optimizing model: onnx/vae.opt.onnx
VAE decoder: original .. 670 nodes, 811 tensors, 1 inputs, 1 outputs
VAE decoder: cleanup .. 670 nodes, 811 tensors, 1 inputs, 1 outputs
[I] Folding Constants | Pass 1
[I]     Total Nodes | Original:   670, After Folding:   499 |   171 Nodes Folded
[I] Folding Constants | Pass 2
2023-04-17 12:54:36.521102014 [W:onnxruntime:, unsqueeze_elimination.cc:20 Apply] UnsqueezeElimination cannot remove node /decoder/mid_block/attentions.0/Unsqueeze_29
2023-04-17 12:54:36.521357396 [W:onnxruntime:, unsqueeze_elimination.cc:20 Apply] UnsqueezeElimination cannot remove node /decoder/mid_block/attentions.0/Unsqueeze_26
2023-04-17 12:54:36.521405920 [W:onnxruntime:, unsqueeze_elimination.cc:20 Apply] UnsqueezeElimination cannot remove node /decoder/mid_block/attentions.0/Unsqueeze_31
2023-04-17 12:54:36.521569614 [W:onnxruntime:, unsqueeze_elimination.cc:20 Apply] UnsqueezeElimination cannot remove node /decoder/mid_block/attentions.0/Unsqueeze_1
[I]     Total Nodes | Original:   499, After Folding:   470 |    29 Nodes Folded
[I] Folding Constants | Pass 3
[I]     Total Nodes | Original:   470, After Folding:   470 |     0 Nodes Folded
VAE decoder: fold constants .. 470 nodes, 774 tensors, 1 inputs, 1 outputs
VAE decoder: shape inference .. 470 nodes, 774 tensors, 1 inputs, 1 outputs
VAE decoder: finished .. 470 nodes, 774 tensors, 1 inputs, 1 outputs
Building TensorRT engine for onnx/clip.opt.onnx: engine/clip.plan
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:604] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:81] The total number of bytes read was 782909211
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:604] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:81] The total number of bytes read was 782909211
[W] onnx2trt_utils.cpp:374: Your ONNX model has been generated with INT64 weights, while TensorRT does not natively support INT64. Attempting to cast down to INT32.
[I]     Configuring with profiles: [Profile().add('input_ids', min=(1, 77), opt=(1, 77), max=(16, 77))]
[I] Building engine with configuration:
    Flags                  | [FP16]
    Engine Capability      | EngineCapability.DEFAULT
    Memory Pools           | [WORKSPACE: 18250.38 MiB, TACTIC_DRAM: 22579.88 MiB]
    Tactic Sources         | []
    Profiling Verbosity    | ProfilingVerbosity.DETAILED
    Preview Features       | [DISABLE_EXTERNAL_TACTIC_SOURCES_FOR_CORE_0805]
[W] kFASTER_DYNAMIC_SHAPES_0805 preview feature is disabled.
[W] TensorRT encountered issues when converting weights between types and that could affect accuracy.
[W] If this is not the desired behavior, please modify the weights or retrain with regularization to adjust the magnitude of the weights.
[W] Check verbose logs for the list of affected weights.
[W] - 225 weights are affected by this issue: Detected subnormal FP16 values.
[W] - 1 weights are affected by this issue: Detected values less than smallest positive FP16 subnormal value and converted them to the FP16 minimum subnormalized value.
[W] - 1 weights are affected by this issue: Detected finite FP32 values which would overflow in FP16 and converted them to the closest finite FP16 value.
[I] Finished engine building in 321.607 seconds
[I] Saving engine to engine/clip.plan
Building TensorRT engine for onnx/unet.opt.onnx: engine/unet.plan
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:604] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:81] The total number of bytes read was 1733934759
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:604] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:81] The total number of bytes read was 1733934759
[W] onnx2trt_utils.cpp:400: One or more weights outside the range of INT32 was clamped
[I]     Configuring with profiles: [Profile().add('sample', min=(2, 4, 64, 64), opt=(2, 4, 64, 64), max=(32, 4, 64, 64)).add('encoder_hidden_states', min=(2, 77, 1024), opt=(2, 77, 1024), max=(32, 77, 1024)).add('timestep', min=[1], opt=[1], max=[1])]
[I] Building engine with configuration:
    Flags                  | [FP16]
    Engine Capability      | EngineCapability.DEFAULT
    Memory Pools           | [WORKSPACE: 18250.38 MiB, TACTIC_DRAM: 22579.88 MiB]
    Tactic Sources         | []
    Profiling Verbosity    | ProfilingVerbosity.DETAILED
    Preview Features       | [DISABLE_EXTERNAL_TACTIC_SOURCES_FOR_CORE_0805]
Segmentation fault (core dumped)

## Environment

**TensorRT Version**: 8.6
**NVIDIA GPU**: A10
**NVIDIA Driver Version**: 525.60.13
**CUDA Version**: 12.0  
**CUDNN Version**: 
**Operating System**: Ubuntu 22.04.1 LTS with nvidia/pytorch:23.02-py3 
**Python Version (if applicable)**: Python 3.8.10
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**:

**Comment by zerollzeng** [comment]
Will check later on A10, @nvpohanh it's caused by OOM?

**Comment by HeKun-NVIDIA** [comment]
Hi
I get the same issue on A10

**Comment by HeKun-NVIDIA** [comment]
> Will check later on A10, @nvpohanh it's caused by OOM?

Run successfully with --build-static-batch

**Comment by zerollzeng** [comment]
Could you try with `--build-preview-features`?

**Comment by ttyio** [comment]
closing since no activity for more than 3 weeks, pls reopen if you still have question, thanks!
